info.toml

```diff
@@ -52,7 +52,8 @@ because we want to assign a different typed value to an existing variable. Somet
 you may also like to reuse existing variable names because you are just converting
 values to different types like in this exercise.
 Fortunately Rust has a powerful solution to this problem: 'Shadowing'!
-You can read more about 'Shadowing' in the book's section 'Variables and Mutability'.
+You can read more about 'Shadowing' in the book's section 'Variables and Mutability': 
+https://doc.rust-lang.org/book/ch03-01-variables-and-mutability.html#shadowing.
```

Comment:
One last minor nit: in my terminal, the final dot messes with the link, so it ends up not jumping to the "Shadowing" section.

Reply:
Removed "."
Also, `cargo` updated the Cargo.lock as well. At this point, I am not sure if it is unwanted.